sapp/ui/issues.py

```diff
@@ -462,26 +476,100 @@ def where_filter(self, filter_instance: Filter) -> "Instance":
             .where_status_is_any_of(filter_instance.statuses)
         )
 
+        if filter_instance.callables is not None and len(filter_instance.callables) > 0:
+            if type(filter_instance.callables) is not list:
```

Comment:
I'm confused my this. Above, `callables` is typed as a `List[Dict[...`, and before it was a `List[str]`. Shouldn't this branch never be hit? Based on the code below, it looks like you expect it to be a `Dict`. If that's the case, I think you need to update the typing on `Filter`

Reply:
It was actually to be `Dict`. Updated the typing on Filter :)

Reply:
ditto: would use `isinstance`